Popup/Popup.js

```diff
@@ -41,6 +41,7 @@ const PopupBase = kind({
 			PropTypes.element,
 			PropTypes.arrayOf(PropTypes.element)
 		]),
+		center: PropTypes.bool,
```

Comment:
We need the documentation for the `center` prop that you added like Button's `animateOnRender ` prop (https://github.com/enactjs/agate/blob/develop/Button/Button.js#L56).

Until now we did not add the documentation properly because we didn't release Agate 1.0 yet. But I think that it's time to prepare the Agate 1.0 from now. So let's add it when we add a new prop.

We might need to cleanup all props of the all component in the Agate in a separate issue.